test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -151,6 +154,23 @@
         StreamingInboundHandler.trackInboundHandlers();
     }
 
+    @Override
+    public boolean isLogsSupported()
+    {
+        return true;
+    }
+
+    @Override
+    public LogAction logs()
+    {
+        // ./build/test/logs/${cassandra.testtag}/${suitename}/${cluster_id}/${instance_id}/system.log
```

Comment:
nit: add reference to the `test/conf/logback-dtest.xml` file so that it is more clear to know where the path comes from